contribs/gmf/apps/desktop/index.html

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html lang="{{mainCtrl.lang}}" ng-app="app" ng-controller="DesktopController as mainCtrl">
+<html lang="{{mainCtrl.lang}}" ng-app="AppDesktop" ng-controller="DesktopController as mainCtrl">
```

Comment:
@pfirpfel This should probably be done on all the others applications :-)

Reply:
ok, I will do this